vault/logical_system_namespaces.go

```diff
@@ -290,6 +290,15 @@ func (b *SystemBackend) handleNamespacesSet() framework.OperationFunc {
 			}
 		}
 
+		nsSealKeyShares, err := b.Core.sealManager.InitializeBarrier(ctx, entry)
+		if err != nil {
+			return logical.ErrorResponse(fmt.Sprintf("%s", err.Error())), err
+		}
+
+		for i, keyShare := range nsSealKeyShares {
+			fmt.Println("Key Share", i, ":", fmt.Sprintf("%x", keyShare))
+		}
+
```

Comment:
This needs to be moved into the `if len(sealConfigs) > 0 {` block, otherwise we panic if we try to create a "normal" namespace.